lib/pg_query/deparse.rb

```diff
@@ -251,7 +251,8 @@ def deparse_funccall(node)
       args << '*' if node['agg_star']
 
       name = (node['funcname'] - ['pg_catalog']).join('.')
-      output << format('%s(%s)', name, args.join(', '))
+      distinct = node['agg_distinct'] ? 'distinct ' : ''
```

Comment:
I think we should use uppercase `DISTINCT` here, since thats what the Postgres docs suggest: http://www.postgresql.org/docs/9.4/static/sql-expressions.html#SYNTAX-AGGREGATES